core/src/main/java/nl/nn/adapterframework/jta/btm/BtmConnectionFactoryFactory.java

```diff
@@ -43,7 +43,7 @@ public BtmConnectionFactoryFactory() {
 	}
 
 	@Override
-	protected ConnectionFactory augment(ConnectionFactory connectionFactory, String connectionFactoryName) {
+	protected ConnectionFactory augmentConnectionFactory(ConnectionFactory connectionFactory, String connectionFactoryName) {
```

Comment:
Why does overriding `augment` not work? Why instead is it made `final` in the super-class and delegating to this new method that is overridden here?

Reply:
Sounds familiar 🙈 

😅